packages/server/modules/core/services/tokens.js

```diff
@@ -2,12 +2,13 @@
 const bcrypt = require('bcrypt')
 const crs = require('crypto-random-string')
 const knex = require('@/db/knex')
+const { ServerAcl: ServerAclSchema } = require('@/modules/core/dbSchema')
```

Comment:
Only reason I'm doing this is to increase coverage, otherwise this new code only gets used in a CLI command which doesn't get tested